rest/server_context.go

```diff
@@ -175,11 +175,17 @@ func (sc *ServerContext) Close(ctx context.Context) {
 		base.InfofCtx(ctx, base.KeyAll, "Couldn't stop stats logger: %v", err)
 	}
 
+	// stop the config polling
 	err = base.TerminateAndWaitForClose(sc.BootstrapContext.terminator, sc.BootstrapContext.doneChan, serverContextStopMaxWait)
 	if err != nil {
 		base.InfofCtx(ctx, base.KeyAll, "Couldn't stop background config update worker: %v", err)
 	}
 
+	// close cached bootstrap bucket connections
```

Comment:
Should this be inside sc.lock.Lock(), to avoid races calling Connection.Close()?